proteinworkshop/datasets/pdb_dataset.py

```diff
@@ -44,11 +46,15 @@ def __init__(
         self.remove_pdb_unavailable = remove_pdb_unavailable
         self.min_length = min_length
         self.max_length = max_length
-        self.split_sizes = split_sizes
+        assert sum(train_val_test) == 1, f"train_val_test need to sum to 1, but sum to {sum(train_val_test)}"
+        self.train_val_test = train_val_test
+        self.split_type = split_type
+        self.split_sequence_similarity = split_sequence_similiarity
+        self.splits = ["train", "val", "test"]
 
     def create_dataset(self):
         log.info(f"Initializing PDBManager in {self.path}...")
-        pdb_manager = PDBManager(root_dir=self.path)
+        pdb_manager = PDBManager(root_dir=self.path, splits=self.splits, train_val_test=self.train_val_test)
```

Comment:
careful, I think the PDBManager arg is still `split_ratios`

Reply:
Agreed